plugins/plugin-grid/src/lib/cell.ts

```diff
@@ -28,7 +28,7 @@ export const renderCell = (
   returnTo: string,
   cell: HTMLElement,
   activation: Record<string, any>, // eslint-disable-line @typescript-eslint/no-explicit-any
-  isFailure: boolean = !activation.response.success,
```

Comment:
this reduces type coverage; because activation.response is an `any`.

Reply:
But with the negation (`!`), the boolean type is inferred:

<img width="726" alt="Screenshot 2019-08-14 at 13 22 42" src="https://user-images.githubusercontent.com/1007051/63017481-be625d00-be96-11e9-82b3-29c34406d459.png">

Reply:
ok, great!